apps/public/src/app/(general)/bijlage/Attachments.tsx

```diff
@@ -10,36 +10,52 @@ import { postMeldingByMeldingIdAttachment } from 'apps/public/src/apiClientProxy
 
 import { BackLink } from '../_components/BackLink'
 
-import { FileUpload } from './_components/FileUpload'
+import { FileInput } from './_components/FileInput'
+import { FileList } from './_components/FileList'
```

Comment:
We don't need a custom component for this, it's already in [the Design System](https://designsystem.amsterdam/?path=/docs/components-forms-file-list--docs)